lib/Parser/ptree.h

```diff
@@ -600,9 +600,21 @@ struct PnFinally : PnStmt
     ParseNodePtr pnodeBody;
 };
 
+enum JsType : byte 
```

Comment:
nit: maybe make an enum class? otherwise we pollute global namespace, then we can do JsType::Object, etc. instead of t_